### PR TITLE
Warlock Crush Hits more of the Tank now

### PR DESCRIPTION
**Pull request by Wisemonster**
## About The Pull Request
Title, crush will now hit the hitbox of tanks and apcs for 10 damage each.
## Why It's Good For The Game
Consistency and intuitiveness. Will slightly lower the damage of insta-crushes on the tank/apc.
## Changelog
:cl:
balance: Warlock's crush now hits more parts of tanks/APCs, dealing 10 damage per part hit.
/:cl:

### CODE REVIEW

**Comment by lbnesquik** [comment]
Did it only hit the center before?

**Review by Lumipharon — APPROVED** [review]

File: code/modules/mob/living/carbon/xenomorph/castes/warlock/abilities_warlock.dm [hunk 1]
```diff
@@ -403,9 +403,9 @@
 				carbon_victim.apply_damage(PSY_CRUSH_DAMAGE * 1.5, STAMINA, blocked = BOMB)
 				carbon_victim.adjust_stagger(5 SECONDS)
 				carbon_victim.add_slowdown(6)
-			else if(isvehicle(victim))
+			else if(isvehicle(victim) || ishitbox(victim))
 				var/obj/vehicle/veh_victim = victim
```
Comment (Lumipharon) [inline code/modules/mob/living/carbon/xenomorph/castes/warlock/abilities_warlock.dm:407]: this can't be typecast as a vehicle anymore

**Review by Lumipharon — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/xenomorph/castes/warlock/abilities_warlock.dm (hunk 1 above)
Comment (Wisemonster) [reply code/modules/mob/living/carbon/xenomorph/castes/warlock/abilities_warlock.dm:407]: So do you want it to use isarmored?

Comment (Wisemonster) [reply code/modules/mob/living/carbon/xenomorph/castes/warlock/abilities_warlock.dm:407]: Alright, should be fixed now

**Review by Lumipharon — APPROVED** [review]

**Pull request merged**